### PR TITLE
ApiManager reads init_service_configs in server config

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

File: contrib/endpoints/src/api_manager/api_manager_impl.cc [hunk 1]
```diff
@@ -43,9 +46,52 @@ ApiManagerImpl::ApiManagerImpl(std::unique_ptr<ApiManagerEnvInterface> env,
...
     }
+  } else if (global_context_->server_config()->init_service_configs_size() >
+             0) {
+    std::vector<std::pair<std::string, int>> list;
+    for (auto item : global_context_->server_config()->init_service_configs()) {
+      std::ifstream config_file(item.service_config_file_full_path());
+      if (config_file.is_open()) {
+        std::stringstream content;
+        std::string line;
+        while (getline(config_file, line)) {
```
Comment (qiwzhang) [inline contrib/endpoints/src/api_manager/api_manager_impl.cc]: can we find a function to read the whole file in one call?

Comment (mangchiandjjoe) [inline contrib/endpoints/src/api_manager/api_manager_impl.cc]: Replace with "content << config_file.rdbuf();"

**Comment by qiwzhang** [comment]
Please do "remove service_config from api_manager_factory" in another PR.

File: contrib/endpoints/include/api_manager/api_manager.h [hunk 2]
```diff
@@ -102,7 +102,7 @@ class ApiManagerFactory {
   // Create an ApiManager object.
   std::shared_ptr<ApiManager> CreateApiManager(
       std::unique_ptr<ApiManagerEnvInterface> env,
-      const std::string &server_config);
+      const std::string &service_config, const std::string &server_config);
```
Comment (qiwzhang) [inline contrib/endpoints/include/api_manager/api_manager.h]: please do this change in a separate PR.

Comment (mangchiandjjoe) [inline contrib/endpoints/include/api_manager/api_manager.h]: I reverted it. I will create another PR for this.
